packages/public/server/src/module/Entity/src/Entity/Controller/EntityController.php

```diff
@@ -86,7 +86,41 @@ public function unrevisedAction()
             }
         }
 
-        $view = new ViewModel(['revisionsBySubject' => $revisionsBySubject]);
+        return $revisionsBySubject;
+    }
+
+    protected function getReviewHelpLinks()
```

Comment:
I have added two new functions so that `unrevisedAcion()` does not get to big. However it is also possible to merge those functions. I don't know, what the best solution is...